src/visualizer/svg.js

```diff
@@ -15,26 +17,49 @@ const {
     const svgLineSpace = svgWidth * 0.04; // the amount of space between each line, by the width of the svg
     const svgLineWidth = svgWidth * 0.01;
     const svgTextSize = svgLineSpace * 0.5;
-    const RANGEWIDTH = 22;
+    const LSRANGEWIDTH = 22; // Learningstyle rangewith, from -11 to 11 = 22
+    const WAHRANGEWIDTH = 2; // Working At Home rangewith, from -1 to 1 = 2
 
     /**
      * @summary Creates and appends a div element with the svg of the learningstyles
      * @param {object} group the group array with students
-     * @returns {HTMLElement} Html element with the graphical info of the group from the argument
+     * @returns {HTMLDivElement} Html div element with the graphical info of the group from the argument
      */
     function createGroupSvg(group){
-        const svg = document.createElementNS("http://www.w3.org/2000/svg", "svg");
-        const learningStyles = [];
-        for (let learningStyle in group.students[0].criteria.learningStyles) {
-            learningStyles.push(learningStyle);
-        }
-        const svgHeight = 2  * svgLineSpace * learningStyles.length;
-        svg.setAttribute("width", svgWidth);
-        svg.setAttribute("height", svgHeight);
-        svg.appendChild(createRect(0, 0, svgWidth, svgHeight)); // create a lightgrey rect that fills the whole svg to show it in browser.
+        const svgDiv = document.createElement("div");
+        svgDiv.setAttribute("class", "svg");
+        svgDiv.setAttribute("style", "width: " + svgWidth + "px;");
+        // Create LearningStyles:
+        const learningStyleTitle = document.createElement("p");
+        learningStyleTitle.innerText = "Learningstyles";
+        svgDiv.appendChild(learningStyleTitle);
+        svgDiv.appendChild(createLearningStyles(group));
+
+        // Create SubjectPreferences:
+        const subjectTitle = document.createElement("p");
+        subjectTitle.innerText = "Subjectpreferences";
+        svgDiv.appendChild(subjectTitle);
+        svgDiv.appendChild(createSubjectPreference(group));
+
+        // Create WorkingAtHome:
+        const workingAtHomeTitle = document.createElement("p");
+        workingAtHomeTitle.innerText = "Working at home";
+        svgDiv.appendChild(workingAtHomeTitle);
+        svgDiv.appendChild(createWorkingAtHome(group));
+        return svgDiv;
+    }
+
+    /**
+     * @summary Creates and appends a div element with the svg of the learningstyles
+     * @param {object} group the group array with students
```

Comment:
Det er vel en group og ikke et array af groups

Reply:
* @param {object} group the group array with students
->
     * @param {object} group the group object with students